cli/src/main.rs

```diff
@@ -1,116 +1,20 @@
-                        .long("birthday")
-                        .value_name("birthday")
-                        .help("Specify wallet birthday when restoring from seed. This is the earlist block height where the wallet has a transaction.")
-                        .takes_value(true))
-                    .arg(Arg::with_name("server")
-                        .long("server")
-                        .value_name("server")
-                        .help("Lightwalletd server to connect to.")
-                        .takes_value(true)
-                        .default_value(lightclient::DEFAULT_SERVER))
-                    .arg(Arg::with_name("dangerous")
-                        .long("dangerous")
-                        .help("Disable server TLS certificate verification. Use this if you're running a local lightwalletd with a self-signed certificate. WARNING: This is dangerous, don't use it with a server that is not your own.")
-                        .takes_value(false))
-                    .arg(Arg::with_name("recover")
-                        .long("recover")
-                        .help("Attempt to recover the seed from the wallet")
-                        .takes_value(false))
-                    .arg(Arg::with_name("nosync")
-                        .help("By default, zecwallet-cli will sync the wallet at startup. Pass --nosync to prevent the automatic sync at startup.")
-                        .long("nosync")
-                        .short("n")
-                        .takes_value(false))
-                    .arg(Arg::with_name("COMMAND")
-                        .help("Command to execute. If a command is not specified, zecwallet-cli will start in interactive mode.")
-                        .required(false)
-                        .index(1))
-                    .arg(Arg::with_name("PARAMS")
-                        .help("Params to execute command with. Run the 'help' command to get usage help.")
-                        .required(false)
-                        .multiple(true))
-                    .get_matches();
-
+    use clap::{App, Arg};
+    let fresh_app = App::new("Zecwallet CLI");
+    let configured_app = configure_clapapp!(fresh_app);
+    let matches = configured_app.get_matches();
     if matches.is_present("recover") {
         // Create a Light Client Config in an attempt to recover the file.
-        let config = LightClientConfig {
-            server: "0.0.0.0:0".parse().unwrap(),
-            chain_name: "main".to_string(),
-            sapling_activation_height: 0,
-            consensus_branch_id: "000000".to_string(),
-            anchor_offset: 0,
-            no_cert_verification: false,
-            data_dir: None,
-        };
-
-        match LightClient::attempt_recover_seed(&config) {
-            Ok(seed) => println!("Recovered seed: '{}'", seed),
-            Err(e)   => eprintln!("Failed to recover seed. Error: {}", e)
-        };
+        attempt_recover_seed();
```

Comment:
Why is this section deleted? The recovery attempt should use the blank config because it can't connect to the server during recovery. Please don't delete this section

Reply:
Sorry, I moved it _into_ the definition of "attempt_recover_seed" so it's been moved, rather than deleted.